types/smallwei__avue/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Type definitions for avue 3.0
+// Project: https://github.com/nmxiaowei/avue
+// Definitions by: Lyoko <https://github.com/SoulLyoko>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference lib="dom" />
+/// <reference types="./element-plus" />
+/// <reference types="./dic" />
+/// <reference types="./form" />
+/// <reference types="./crud" />
+/// <reference types="./tree" />
+/// <reference types="./global" />
```

Comment:
these don't use module name resolution. I think they should all be `<reference path="./element-plus" />` and so on. I think that would let you get rid of OTHER_FILES.txt too.